.github/workflows/ci.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release? yes/no'
+        default: 'no'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Cache Gradle
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle
+          key: ${{ runner.os }}-${{ hashFiles('gradle') }}
+      - name: Fix Docker-In-Docker Mounts
+        run: |
+          sudo systemctl stop docker
+          echo '{"cgroup-parent":"/actions_job","storage-driver":"vfs"}' | sudo tee /etc/docker/daemon.json
```

Comment:
I was trying to understand this fix based on your commit message and went on the internet. I found this issue: https://github.com/docker/for-linux/issues/1138 - is it connected? I can see @dagguh activity there. If it's connected then maybe it's worth adding to the commit message for reference in case the problem would be resolved in the future and we could try removing this storage driver override.

Reply:
Actually it came from https://github.com/moby/moby/issues/13742#issuecomment-725197223

- [ ] link the workaround source